docs/guides/faq.md

```diff
@@ -0,0 +1,120 @@
+# FAQ
+
+## Q: What is video.js?
+video.js is an extendable wrapper around the native video element. It does the following:
+  * Offers a plugin API so that different types of video can be handed to the native
+    video element. IE:  HLS, Flash, or HTML5 video.
+  * Unifies the native video api across browsers (polyfilling support for features
+  * if necessary)
+  * Offers an extendable and themable UI
+  * Takes care of accessibility for the user (in-progress)
+
+## Q: How do I install video.js?
+Currently video.js can be installed using bower, npm, or by serving a release file from
+a github tag. For information on doing any of those see the [install guide](http://videojs.com/getting-started/).
+
+## Q: What do video.js version numbers mean?
+video.js follows [semver](http://semver.org/) which means that the API should not change
+out from under a userunless there is a major version increase.
+
+## Q: How can I troubleshoot playback issues?
+See the [troubleshooting guide](/docs/guides/troubleshooting.md) for information on troubleshooting playback issues
+
+## Q: A video does not play in a specific browser. Why?
+Chrome and Safari both rely on byte-range requests for playing back video. So It is important to
+Make sure that the location which is serving your video files supports byte-range requests.
+Often PHP serving video is an issue as it does not support byte-range requests by default.
+
+## Q: Why does the entire video download before playback? Why does the video load for a long time?
+This could mean that your video is not encoded with the metadata -- things like duration,
+video dimensions, etc -- at the beginning of the video. This means that the browser will
+need to load the entire video before it know how long the video is and what to do with
+the video.
+
+## Q: I see an error thrown that mentions `vdata12345`. What is that?
+This error is thrown when an element that is associated with a component is removed
+from the DOM but the event handlers associated with the element are not removed. This
+is almost always due to event listeners not being disposed when dispose is called on
+a component.
```

Comment:
this is still up in the air, but I think we may make a troubleshooting guide and link this question there when we have it.